### DISCUSSION TITLE
Bug or am I being stupid?

### DISCUSSION THREAD

**Discussion by ItsMeRitch** [blakeblackshear/frigate#13508] [Report a Bug] [answered]
Hi folks,

When viewing my exports page from mobile (iOS 17 Safari), I can't seem to delete any of them. The option appears to be behind the thumbnail and is disabled. 

![image](https://github.com/user-attachments/assets/6d6aa488-81a3-44ff-9540-ae4247c9aa80)

**Comment by NickM-27** [comment, marked as answer]
This is fixed in 0.14.1
